waitress/app/utils.py

```diff
@@ -128,98 +130,82 @@ def difference(cls, repo_users, db_users):
         A method that gets the difference between users in the system and
         users in slack group.
         """
-        users = {}
         unsaved_users = []
```

Comment:
No need for unsaved users anymore as the variable is no longer being used

Reply:
Removed. That was an oversight.